cmd/raw.go

```diff
@@ -54,19 +56,30 @@ func updateClientCmd() *cobra.Command {
 			if err = chains[src].AddPath(args[2], dcon, dcha, dpor, dord); err != nil {
 				return err
 			}
+
+			height, err := cmd.Flags().GetInt64(flags.FlagHeight)
 			if err != nil {
 				return err
 			}
 
-			dstHeader, err := chains[dst].UpdateLiteWithHeader()
-			if err != nil {
-				return err
+			var dstHeader *tmclient.Header
+
+			if height > 0 {
+				dstHeader, err = chains[dst].UpdateLiteWithHeaderHeight(int64(height))
+				if err != nil {
+					return err
+				}
+			} else {
+				dstHeader, err = chains[dst].UpdateLiteWithHeader()
+				if err != nil {
+					return err
+				}
 			}
 
 			return sendAndPrint([]sdk.Msg{chains[src].PathEnd.UpdateClient(dstHeader, chains[src].MustGetAddress())}, chains[src], cmd)
 		},
 	}
-	return cmd
+	return liteFlags(cmd)
```

Comment:
Note: We just want the height flag here, not the hash or URL flags.

Reply:
Fixed.

Reply:
You are the man! Thank you!